common/lib/xmodule/xmodule/modulestore/mongo/base.py

```diff
@@ -883,7 +883,8 @@ def _cache_children(self, course_key, items, depth=0):
         apply_cached_metadata=bool,
         using_descriptor_system="None|CachingDescriptorSystem"
     )
-    def _load_item(self, course_key, item, data_cache, apply_cached_metadata=True, using_descriptor_system=None):
+    def _load_item(self, course_key, item, data_cache,
+                   apply_cached_metadata=True, using_descriptor_system=None, for_parent=None):
```

Comment:
Docstring update for for_parent? may be in `def load_item` too? https://github.com/cpennington/edx-platform/blob/pass-xblock-parents/common/lib/xmodule/xmodule/modulestore/mongo/base.py#L226